lib/auth/auth_with_roles.go

```diff
@@ -201,6 +201,17 @@ func (a *ServerWithRoles) hasBuiltinRole(name string) bool {
 	return HasBuiltinRole(a.context.Checker, name)
 }
 
+// hasBuildinRoleInList checks that the attached checker is a BuiltinRoleSet
+// and whether any of the given roles match the role set.
+func (a *ServerWithRoles) hasBuiltinRoleInList(roles ...types.SystemRole) bool {
```

Comment:
Maybe:

```suggestion
func (a *ServerWithRoles) hasBuiltinRole(roles ...types.SystemRole) bool {
```

so it reads a bit more naturally?